chart/values.yaml

```diff
@@ -3,6 +3,7 @@
 # Declare variables to be passed into your templates.
 
 global:
+  imageRegistry: public.ecr.aws/k4y9r6y5
```

Comment:
lets keep the default in the charts to docker and set it to ECR only in the mission-control chart